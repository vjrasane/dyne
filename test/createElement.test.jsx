import Dyne from "../src";

describe("createElement", () => {
  it("creates static element", () => {
    const Component = <div>Hello World!</div>;
    const dom = <Component />;
    expect(dom).toMatchSnapshot();
  });
});
