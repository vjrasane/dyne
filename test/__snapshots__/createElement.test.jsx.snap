// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createElement creates static element 1`] = `
Object {
  "children": Array [
    "Hello World!",
  ],
  "props": Object {},
  "type": "div",
}
`;
